README.rst

```diff
@@ -1,102 +1,133 @@
 
-How To Contribute
------------------
+.. code:: bash
 
-Contributions are very welcome.
+    pip install -e
 
-Please read `How To Contribute <https://github.com/openedx/.github/blob/master/CONTRIBUTING.md>`_ for details.
 
+To run the test suite:
 
-PR description template should be automatically applied if you are sending PR from github interface; otherwise you
-can find it it at `PULL_REQUEST_TEMPLATE.md <https://github.com/openedx/completion/blob/master/.github/PULL_REQUEST_TEMPLATE.md>`_
+.. code:: bash
 
-Issue report template should be automatically applied if you are sending it from github UI as well; otherwise you
-can find it at `ISSUE_TEMPLATE.md <https://github.com/openedx/completion/blob/master/.github/ISSUE_TEMPLATE.md>`_
+    pip install tox
+    tox # to run only a single environment, do e.g. tox -e py38-django42-drflatest
 
-Reporting Security Issues
--------------------------
 
-Please do not report security issues in public. Please email security@openedx.org.
+To use a Django shell to test commands:
 
-Using with Docker Devstack
---------------------------
+.. code:: bash
 
-Prerequisite: Have your Open edX `Devstack <https://github.com/openedx/devstack>`_ properly installed.
+    make install
+    python manage.py migrate
+    python manage.py shell
+    >>> from completion.models import BlockCompletion
+    >>> <other commands...>
 
-Note: When you see "from inside the lms" below, it means that you've run ``make lms-shell`` from your devstack
-directory and are on a command prompt inside the LMS container.
 
-#. Clone this repo into ``../src/`` directory (relative to your "devstack" repo location). This will mount the
-   directory in a way that is accessible to the lms container.
+Deploying
+*********
 
-#. From inside the lms, uninstall completion and reinstall your local copy. You can just copy the following line::
+Tagged versions of the completion library are released to pypi.org.
 
-    pip uninstall completion -y; pip install -e /edx/src/completion/
+To use the latest release in your project, add the following to your pip requirements file:
 
-#. Now, get your completion development environment set up::
+.. code:: bash
 
-    cd /edx/src/completion
-    virtualenv completion-env
-    source completion-env/bin/activate
-    make install
+    edx-completion
 
-#. Don't forget to enable the waffle switch as described above in "Enabling in the LMS"
+Getting Help
+************
 
-#. That's it!  In order to simulate a given tox environment ``(django18, django111, quality)``, run ``tox -e <env>`` for the env in question.  If you want to run ``pytest`` directly::
+Documentation
+=============
 
-    pytest completion/tests/test_models.py
+Start by going through `the documentation`_ (generated from `/docs </docs/index.rst>`_).  If you need more help see below.
 
-Getting Help
-------------
+.. _the documentation: https://docs.openedx.org/projects/completion
```

Comment:
This link doesn't work for me

Reply:
Ya -- I need someone with access please add them? Procedure noted here: https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs

(same issue for edx-submissions, btw)

Reply:
Hmm, that page is archived and I don't know the current procedure. @feanil can you help out here?